flow/core/kernel/network/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""Empty init file to ensure documentation for the network is created."""
+
+from flow.core.kernel.network.base import KernelNetwork
```

Comment:
This is probably irrelevant to this PR, but maybe we should rename the `KernelNetwork` to `BaseKernelNetwork`, and `TraCINetwork` ro `TraCIKernelNetwork`

Reply:
This makes these 3 kerne names uniform.

Reply:
done